src/node_file.cc

```diff
@@ -581,6 +581,25 @@ static void InternalModuleStat(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(rc);
 }
 
+static void InternalModuleRealpath(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK(args[0]->IsString());
+  node::Utf8Value path(env->isolate(), args[0]);
+
+  uv_fs_t req;
+  int rc = uv_fs_realpath(env->event_loop(), &req, *path, nullptr);
+  if (rc == 0) {
+    const char* r = static_cast<const char*>(req.ptr);
+    Local<String> str = String::NewFromUtf8(env->isolate(),
```

Comment:
Just noting: this may have issues on linux filesystem's if the path is not UTF8 encoded